src/main/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -55,13 +55,13 @@
     private String likelihoodsAsString_PLs = null;
 
     /**
-     * initialize num likelihoods cache
+     * initialize num likelihoods cache to up to 5 alleles and 10 ploidy
```

Comment:
you can reference the constants in the javadoc so it doesn't accidentally get out of date.

Reply:
Oh true. I'll move the cache initialization to the cache class constructor and reference them there